blazor-features/RequiredParametersInBlazor/RequiredParametersInBlazorTests/UnitTest1.cs

```diff
@@ -0,0 +1,30 @@
+﻿using RequiredParametersInBlazor;
+using RequiredParametersInBlazor.Components;
+
+namespace RequiredParametersInBlazor.Tests
+{
+    public class AgeCardTests : TestContext
+    {
+        [Fact]
+        public void WhenYearOfBirthIsSupplied_ThenAgeIsReturned()
```

Comment:
Should be `Given_When_Then` format.

Reply:
Implemented Given_When_Then for the method name